src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderServiceTest.java

```diff
@@ -133,13 +133,22 @@ void enumerateSnapshotRequestsByDatasetId() {
   public void getConceptChildren(CloudPlatform cloudPlatform) {
     Dataset dataset = makeDataset(cloudPlatform);
     when(datasetService.retrieve(dataset.getId())).thenReturn(dataset);
-    var concepts = List.of(new SnapshotBuilderConcept().name("concept1").id(1));
-    mockRunQueryForConcepts(cloudPlatform, concepts, dataset);
+
+    SnapshotBuilderDomainOption domainOption = new SnapshotBuilderDomainOption();
+    domainOption.name("domainId").tableName("domainTable").columnName("domain_concept_id");
```

Comment:
I was wondering why this needs so much mocked to work; once https://broadworkbench.atlassian.net/browse/DC-952 is fixed this will get simpler.